pkgs/development/python-modules/bezier/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  numpy,
+
+  cmake,
+  gfortran,
+
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  name = "bezier";
+  version = "2024.6.20";
+  src = fetchFromGitHub {
+    owner = "dhermes";
+    repo = "bezier";
+    tag = version;
+    hash = "sha256-TH3x6K5S3uV/K/5e+TXCSiJsyJE0tZ+8ZLc+i/x/fV8=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ numpy ];
```

Comment:
Please add the optional dependencies as well. bezier has this [list of optional dependencies](https://github.com/dhermes/bezier/blob/main/setup.py#L79), and this is the [nixpkgs documentation for adding optional-dependencies](https://nixos.org/manual/nixpkgs/unstable/#python-optional-dependencies).